interfaces/insight.ts

```diff
@@ -1,16 +1,20 @@
-import {FiledRequest} from "./filedRequest";
-import {User} from "./user";
 
 export const INSIGHTS_COLLECTION = "insights"
 
 export interface Insight {
     createdBy: string; // userId
     createDate: number;
+    lastUpdated: number;
```

Comment:
This might be a datetime type

Reply:
For database purposes, the dates would ideally be in timestamp format. If it becomes a hassle to add conversions for wherever the dates are used, I can add a singular point of conversion for reading/writing to firestore